### PR TITLE
Adding Google Analytics to the panel view

### PR DESCRIPTION
**Pull request by dfarache**

### CODE REVIEW

File: app/chrome/panel.html [hunk 1]
```diff
@@ -3,14 +3,16 @@
...
-  <meta http-equiv="Content-Security-Policy" content="default-src *; style-src 'self' 'unsafe-inline'; script-src 'self' 'unsafe-inline' 'unsafe-eval' https://maps.googleapis.com ">
+  <meta http-equiv="Content-Security-Policy" content="default-src *; style-src 'self' 'unsafe-inline'; script-src 'self' 'unsafe-inline' 'unsafe-eval' https://ssl.google-analytics.com https://maps.googleapis.com https://www.googletagmanager.com https://www.google-analytics.com">
   <link rel="stylesheet" href="style.css" media="all">
   <!--
       - JavaScript and HTML must be in separate files: see our Content Security
       - Policy documentation[1] for details and explanation.
       -
       - [1]: https://developer.chrome.com/extensions/contentSecurityPolicy
     -->
+    <!-- Global site tag (gtag.js) - Google Analytics -->
```
Comment (ddebasilio) [inline app/chrome/panel.html]: There is no "gtag.js" file in the project, is this just an indication for something else? or should it read "analytics.js"?

File: app/chrome/manifest.json [hunk 2]
```diff
@@ -23,8 +23,7 @@
 
   "offline_enabled": true,
 
-  "content_security_policy": "script-src 'self' 'unsafe-eval' 'unsafe-inline' https://maps.googleapis.com; object-src 'self' 'unsafe-eval' 'unsafe-inline'",
-
+  "content_security_policy": "script-src 'self' https://www.googletagmanager.com https://maps.googleapis.com https://www.google-analytics.com https://ssl.google-analytics.com; object-src 'self'",
```
Comment (carocad) [inline app/chrome/manifest.json]: @dfarache why did you remove the `unsafe-eval` and `unsafe-inline`? or rather did you tested the complete simulator to see if it works like that?

Comment (dfarache) [reply app/chrome/manifest.json]: I removed unsafe-inline because it seems to be considered unsafe and ignored by Chrome as you may see in the screenshot below.

Then, unsafe-eval does not seem to be needed: our external dependencies (maps and analytics) load correctly without it and it's probably better to disallow it unless it's strictly needed.

screen shot 2018-08-20 at 13 34 18

Comment (carocad) [reply app/chrome/manifest.json]: @dfarache the simulator does a lot of client side "eval" so I would prefer if we properly test this before deploying it. 

I have a meeting today with @dkkeller about the test cases for the simulator so it would be good if you could join us
